src/notifications.js

```diff
@@ -12,7 +12,7 @@ class Notifications extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const {notifications} = nextProps;
+    const notifications = nextProps.notifications.toJS();
```

Comment:
the change to `componentWillReceiveProps` is agnostic to whether the rest of the app is using `immutable` elsewhere or not, since we're directly `connect`ing `state.notifications` to the component, i.e. (https://github.com/gor181/react-notification-system-redux/blob/master/example/src/components/container.js#L62).    If a consumer follows the docs and examples, `nextProps.notifications` will be always be the immutable `List`.